2_process/src/create_site_list.R

```diff
@@ -1,4 +1,4 @@
-create_site_list <- function(wqp_data,nwis_sites,nwis_daily_data,nwis_inst_data,hucs,crs_out="NAD83",fileout){
+create_site_list <- function(wqp_data,nwis_sites,nwis_daily_data,nwis_inst_data,hucs,crs_out="NAD83"){
```

Comment:
We can delete the description for "@param fileout" below. I think this change to an object target makes sense since it's more or less replaced by `p2_sites_wi_segs`. I haven't tried running the code yet - I wonder if we'll need to make any changes to the .Rmd file for the pipeline to build?

Reply:
Line 20 in the .Rmd file should be changed to:

`site_list_path <- tar_read(p2_sites_w_segs_csv)`